statsd/client.py

```diff
@@ -50,7 +50,7 @@ def stop(self, send=True):
         if self._start_time is None:
             raise RuntimeError('Timer has not started.')
         dt = time.time() - self._start_time
-        self.ms = int(round(1000 * dt))  # Convert to milliseconds.
+        self.ms = float(1000 * dt)  # Convert to milliseconds.
```

Comment:
@scbunn The ``float()`` call is not necessary, ``dt`` is already a float and ``1000 * dt`` will be a float too.